pythonforandroid/recipes/regex/__init__.py

```diff
@@ -3,10 +3,15 @@
 
 class RegexRecipe(CompiledComponentsPythonRecipe):
     name = 'regex'
-    version = '2017.07.28'
-    url = 'https://pypi.python.org/packages/d1/23/5fa829706ee1d4452552eb32e0bfc1039553e01f50a8754c6f7152e85c1b/regex-{version}.tar.gz'
+    version = '2019.06.08'
+    url = (
+        'https://files.pythonhosted.org/packages/6f/4e/'
```

Comment:
I'm not sure if we should split the url line as I did, to keep us below 80 characters (I think that We already did that in some recipes...:thinking:...I don't remember now)...anyway...We could also use a `noqa` marker instead.

Reply:
Anyway I would not put that has since it's often changing. There's a hack to make it work, try that:
```python
url = 'https://pypi.python.org/packages/source/r/regex/regex-{version}.tar.gz'
```